packages/actor-init-sparql-file/bin/http.ts

```diff
@@ -1,3 +1,8 @@
 #!/usr/bin/env node
 import {HttpServiceSparqlEndpoint} from "@comunica/actor-init-sparql";
-HttpServiceSparqlEndpoint.runArgsInProcess(__dirname + '/../', __dirname + '/../config/config-default.json');
+
+const defaultConfigPath = __dirname + '/../config/config-default.json';
+const configResourceURL = process.env.COMUNICA_CONFIG ? process.env.COMUNICA_CONFIG : defaultConfigPath;
+
+HttpServiceSparqlEndpoint.runArgsInProcess(process.argv.slice(2), process.stdout, process.stderr,
```

Comment:
Now that I look at it, you may actually be able to abstract this slightly more:

You can pass both `defaultConfigPath` and `process.env` to `runArgsInProcess` instead of `configResourceURL`, and do the check of `process.env.COMUNICA_CONFIG` (and calculation of `configResourceURL`) inside this function.

Reply:
Wasn't it originally the idea to keep calls to process out of HttpServiceSparqlEndpoint?